src/cacheHttpClient.ts

```diff
@@ -82,12 +83,11 @@ function createHttpClient(): HttpClient {
     );
 }
 
-export function getCacheVersion(): string {
+export function getCacheVersion(useZstd?: boolean): string {
```

Comment:
This function is a ["boolean-trap"](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap). It might be worth considering using an enum here, like:

```ts
enum CompressionTool {
	Zstd,
	GnuTar
}
```

Reply:
Well, in my opinion, this is not a big deal. The enum has mistakes, depending on how you look at it, and should not be actively recommended for use. If you want to avoid use boolean, you should use string literal types combine with union types. But it's too overkill for this purpose. So, there is no problem with the current code.

Reply:
Never said that it is a big deal :)

I was more coming from a readability-PoV because the first change I saw in the PR diff was:

```ts
test("getCacheVersion with zstd compression returns version", async () => {
    testUtils.setInput(Inputs.Path, "node_modules");

    const result = getCacheVersion(true);

    expect(result).toEqual(
        "273877e14fd65d270b87a198edbfa2db5a43de567c9a548d2a2505b408befe24"
    );
});
```

and I didn't understand what `getCacheVersion(true)` means, hence the comment about boolean-traps :)

Reply:
Sorry. This kind of expression is really common in Japanese, So I said. (English is still difficult for me...) It certainly seems to have a readability problem, but I think it's basically within the acceptable range.

Reply:
If this needs to be changed, I suggest a major redesign, but it requires too much work and this action probably won't be a large codebase that it needs.

Reply:
PR updated to use enum instead.